conf/application.conf

```diff
@@ -132,7 +132,8 @@ microservice {
       host = "localhost"
       environment = "dev"
       authorisation-token = "dev"
```

Comment:
See https://github.com/hmrc/direct-debit/blob/master/conf/application.conf for an example of this